pkg/apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -85,6 +85,24 @@ type ClusterInstallationSpec struct {
 	// Defines the probe to check if the application is ready to accept traffic.
 	// +optional
 	ReadinessProbe corev1.Probe `json:"readinessProbe,omitempty"`
+
+	// Certificates tells the operator which secret contains certificate PEM file(s).
+	// Each file will be mounted as volume inside the Mattermost container.
+	// +optional
+	Certificates Certificates `json:"certificates,omitempty"`
```

Comment:
maybe is worth explicitly say which type of certificate, in this case i would name it `CACertificates`